documentation/integrating/integrating.md

```diff
@@ -0,0 +1,55 @@
+# Integrating Form Builder with 3rd Party Applications
+
+Any form hosted with the Form Builder can integrate easily with 3rd party applications
+by taking advantage of the [JSON output](../submitting/submitting.md) option.
+
+See the [HMCTS Complaints Adapter](https://github.com/ministryofjustice/hmcts-complaints-formbuilder-adapter)
+for an example of an application that consumes this JSON output and generates cases
+in a 3rd Party CMS system.
+
+## Requirements
+
+There are a few requirements for setting up and then using the JSON output:
+
+### 1. Config vars
+
+Your form needs to be configured in Publisher to enable the JSON output feature. You must set both:
+
+a. `SERVICE_OUTPUT_JSON_ENDPOINT` which is the external endpoint you want JSON sent to.
+b. `SERVICE_OUTPUT_JSON_KEY` which is the 16 byte shared secret used for encryption.
```

Comment:
Need to add a blank line between "a." and "b." or the markdown parser will squash this into one line